src/services/drive/share/driveShare.service.ts

```diff
@@ -62,18 +43,46 @@ class DriveShareService {
     return this.sdk.share.deleteShareLink(shareId);
   }
 
+  stopSharingItem({ itemUUID, itemType }: { itemUUID: string; itemType: 'folder' | 'file' }) {
```

Comment:
we'll need to prepare this service for mobile AFS, nothing to do here, but take this into account.

Anyways, why itemUUID instead of just uuid?

Reply:
Nothing special, it is clearer to me because these objects have a wide range of fields. If you see it redundant because the name of the function already indicates that it is an _item_ we can leave it in **uuid** and **type**.

Reply:
is more like to maintain a property - variable relationship, I prefer to have `stopSharingFile` and `stopSharingFolder` rather than this in any case cause is more explicit about what this is going to do.

Note that we usually don't use the `DriveItem` concept, instead we use `DriveFile` and `DriveFolder` concepts, but I agree that this is not a widely used pattern at Internxt, so this one is up to you, but whatever you decide, let's keep it consistent.